digital_land/pipeline.py

```diff
@@ -327,7 +327,19 @@ def __init__(self, directory=None) -> None:
         self.schema = Schema("lookup")
         self.entity_num_gen = EntityNumGen()
 
-    def add_entry(self, entry):
+    def add_entry(self, entry, is_new_entry=True):
```

Comment:
should 
`is_new_entry` 
actually be 
`validate`
as it's actually turning validation on and off

Reply:
This is just a matter of opinion.
One is a property of the entry, the other is an instruction to the method.
I'd prefer attributing the parameter rather than instructing the method because I'd rather not declare that validation is optional. It's really just a quirk for backward compatibility.
However, if changing it gets it 'passed' then I'll just change it.

Reply:
Happy to leave as is my concern is that it is more difficult to understand. but I do agree it's an opinion!